scripts/run-dev-worker.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -ex
 
-celery -A src worker -l info
\ No newline at end of file
+celery -A src worker -l INFO
```

Comment:
the latest version of celery - 5.0.0 - has been released and now valid log levels are DEBUG, INFO, WARNING, ERROR, CRITICAL, FATAL